### DISCUSSION TITLE
国产开源社群的运营，为何总是画风奇特？ | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#88] [Comments]
# 国产开源社群的运营，为何总是画风奇特？ | 夜天之书

在过去几年的投入和关注下，国产开源社群如雨后春笋一般冒了出来。今天，以 GPT 为首的 AI 新势力接过话题度的接力棒，我们可以在降温周期里回顾一下过去几年间冒出来的国产开源社群都有什么样的成绩，有些什么样共性的问题可以改进。 本文标题是国产开源社群的运营总是画风奇特，这也符合近几年来作为理论上开源社群的主要参与者，即软件开发者们的主流感受。从眼花缭乱的开源营销，到一次性开源、开关式开源的另类创新

https://www.tisonkun.org/2023/06/04/oss-community-in-china/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

随着具体项目的参与, 以及持续的文章输出, 可以感觉到有越来越多的个人风格涌现了...

画风的奇特, 现在感觉主要是 "注意力渗透压" 的物理原因;
中国多数有点儿影响力的开源都是大厂引发的,
而主创人员本身多数还并不能脱离大厂画风奇特的 KPI 抽动,
这就导致开源项目核心成员注意力密度远远高于外部成员,
关键内部对项目的要求, 而无法被外部毫无影响力的要求干挠...
也就是说, 除了形式上具备开源项目的一切特征外,
核心成员的心理状态, 根本没有迁移到 开源工程 的心理进程中,
心口不一, 当然越来越扭曲...
